Formula/proxychains-ng.rb

```diff
@@ -1,9 +1,8 @@
 class ProxychainsNg < Formula
   desc "Hook preloader"
   homepage "https://sourceforge.net/projects/proxychains-ng/"
-  url "https://github.com/rofl0r/proxychains-ng/releases/download/v4.12/proxychains-ng-4.12.tar.xz"
-  sha256 "482a549935060417b629f32ddadd14f9c04df8249d9588f7f78a3303e3d03a4e"
-  revision 1
+  url "https://ftp.barfooze.de/pub/sabotage/tarballs/proxychains-ng-4.13.tar.xz"
```

Comment:
Please use the github tarball: https://github.com/rofl0r/proxychains-ng/archive/v4.13.tar.gz
And squash all commits into one, to avoid conflicts